pkg/test/resourcefixture/testdata/basic/compute/v1beta1/computeforwardingrule/globalcomputeforwardingrule/_http.log

```diff
@@ -793,46 +790,9 @@ X-Xss-Protection: 0
 
 ---
 
-GET https://compute.googleapis.com/compute/v1/projects/${projectId}/global/forwardingRules/computeglobalforwardingrule-${uniqueId}?alt=json
```

Comment:
You _could_ do the same trick where you make an extra (pointless) call to get here, just to keep the log simple.  But we can also just look at it and understand what's happening ... up to you!